github/PullRequest.py

```diff
@@ -525,7 +525,7 @@ def get_review_comment(self, id):
             "GET",
             self._parentUrl(self.url) + "/comments/" + str(id)
         )
-        return github.PullRequestComment.PullRequestComment(self._requester, headers, data, completed=True)
+        return github.PullRequestComment.PullRequestComment(self._requester, headers, data, completed=True)    
```

Comment:
Looks like you're adding some trailing whitespace here, FWIW

Reply:
Ok, I will fix in the next PR. Thanks